composer.json

```
@@ -31,7 +31,8 @@
         "phpstan/extension-installer": "^1.4.1",
         "phpstan/phpstan": "^1.11.4",
         "phpstan/phpstan-phpunit": "^1.4.0",
-        "phpunit/phpunit": "^8.5.38"
+        "phpunit/phpunit": "^8.5.38",
+        "rector/rector": "^1.1"
```

For development dependencies, please let's always use a specific version number:

```suggestion
        "rector/rector": "^1.1.0"
```

---

Fixed it to the suggestesd v1.1.0